### PR TITLE
Spack on Windows documentation

### PR DESCRIPTION
**Pull request by loulawrence**
Add instructions to Getting Started for Spack on Windows

### CODE REVIEW

File: lib/spack/docs/getting_started.rst [hunk 1]
```diff
@@ -1322,3 +1322,69 @@ To ensure that Spack does not autodetect the Cray programming
...
 will cause Spack to treat a linux container on a Cray system as a base
 linux distro.
+
+.. _windows_support:
+
+----------------
+Spack On Windows
+----------------
+
+You will require a MSVC compiler, Git and CMake in order to use Spack on Windows.
```
Comment (BetsyMcPhail) [inline lib/spack/docs/getting_started.rst:1332]: Are git and cmake necessary when using the installer?

File: lib/spack/docs/getting_started.rst [hunk 2]
```diff
@@ -1322,3 +1322,69 @@ To ensure that Spack does not autodetect the Cray programming
...
+      ninja:
+        externals:
+        - spec: ninja@1.8.2
+          prefix: 'c:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja'
+        buildable: False
+
+Spack on Windows should be run through the Spack Package Manager application added by
+the installer, or by running ``spack_cmd.bat`` from the spack installed directory. To
+add detectable compilers and packages, run ``spack compiler find`` and ``spack external find``
+
```
Comment (BetsyMcPhail) [inline lib/spack/docs/getting_started.rst:1380]: Should there be a section on creating the installer? Documentation for that process should probably be somewhere, I'm sure if this document is the appropriate place?

**Review by BetsyMcPhail — CHANGES_REQUESTED** [review]

File: lib/spack/docs/getting_started.rst (hunk 1 above)
Comment (loulawrence) [reply lib/spack/docs/getting_started.rst:1332]: Yes, for right now, you'll need git to pull feature/windows and cmake, I believe isn't buildable from Spack without another cmake to build it. Hopefully this list of requirements gets smaller in later versions.

Comment (BetsyMcPhail) [reply lib/spack/docs/getting_started.rst:1332]: Ah, okay.

**Review by BetsyMcPhail — APPROVED** [review]

**Pull request merged**